src/main/java/org/hydev/mcpm/utils/PluginJarFile.java

```diff
@@ -31,7 +28,7 @@ public PluginJarFile(@NotNull File file, int mode, @NotNull Charset charset) thr
     }
 
     /**
-     * Read the content of a file inside a jar/zip file
+     * Read the content of a file inside a jar/zip file.
```

Comment:
Is this part of the linter or was this staged at the time?

Reply:
My `checkstyle.yml` was modified from Google's `checkstyle.yml`. They originally check for periods at the first line of each javadoc but I removed it because it isn't something important

Reply:
This is a cool feature! I'm surprised the linter would touch comments at all because this has the potential of working weirdly with code blocks and javadoc embeds. Keep it if you can!